notification/method/webpush.php

```diff
@@ -140,10 +146,18 @@ public function notify()
 	{
 		$insert_buffer = new \phpbb\db\sql_insert_buffer($this->db, $this->notification_webpush_table);
 
+		// Load all users data we want to notify
+		$notify_users = $this->load_recipients_data();
+
 		/** @var type_interface $notification */
 		foreach ($this->queue as $notification)
 		{
 			$data = $notification->get_insert_array();
+
+			// Choose receiving user's language
+			$recipient_data = $this->user_loader->get_user($notification->user_id);
+			$this->language->set_user_language($recipient_data['user_lang'], $this->language->get_used_language() !== $recipient_data['user_lang']);
```

Comment:
maybe these instead, so we avoid calling set_user_lang at all if it's not necessary?
```php
$change_user_lang = $this->language->get_used_language() !== $recipient_data['user_lang'];
if ($change_user_lang)
{
   $this->language->set_user_language($recipient_data['user_lang'], true);
}